source/particle/world.cc

```diff
@@ -1328,16 +1335,33 @@ namespace aspect
 
           tracer_weight = prm.get_integer("Tracer weight");
 
-          if (prm.get ("Load balancing strategy") == "none")
-            particle_load_balancing = no_balancing;
-          else if (prm.get ("Load balancing strategy") == "remove particles")
-            particle_load_balancing = remove_particles;
-          else if (prm.get ("Load balancing strategy") == "remove and add particles")
-            particle_load_balancing = remove_and_add_particles;
-          else if (prm.get ("Load balancing strategy") == "repartition")
-            particle_load_balancing = repartition;
-          else
-            AssertThrow (false, ExcNotImplemented());
+          const std::vector<std::string> strategies = Utilities::split_string_list(prm.get ("Load balancing strategy"));
+          particle_load_balancing = ParticleLoadBalancing::no_balancing;
+
+          for (std::vector<std::string>::const_iterator strategy = strategies.begin(); strategy != strategies.end(); ++strategy)
+            {
+              if (*strategy == "remove particles")
+                particle_load_balancing = typename ParticleLoadBalancing::Kind(particle_load_balancing | ParticleLoadBalancing::remove_particles);
+              else if (*strategy == "add particles")
+                particle_load_balancing = typename ParticleLoadBalancing::Kind(particle_load_balancing | ParticleLoadBalancing::add_particles);
+              else if (*strategy == "remove and add particles")
+                particle_load_balancing = typename ParticleLoadBalancing::Kind(particle_load_balancing | ParticleLoadBalancing::remove_and_add_particles);
+              else if (*strategy == "repartition")
+                particle_load_balancing = typename ParticleLoadBalancing::Kind(particle_load_balancing | ParticleLoadBalancing::repartition);
+              else if (*strategy == "none")
+                {
+                  particle_load_balancing = ParticleLoadBalancing::no_balancing;;
```

Comment:
oh, while not wrong, the second semicolon is still unnecessary. please remove before merging.